ui/apps/platform/src/services/AuthService/AuthService.ts

```diff
@@ -76,9 +76,9 @@ export type AuthProvider = {
     active?: boolean;
     groups?: Group[];
     defaultRole?: string;
-    requiredAttributes: AuthProviderRequiredAttributes[];
+    requiredAttributes: AuthProviderRequiredAttribute[];
     traits?: Traits;
-    claimMappings: Record<string, string>;
+    claimMappings: Record<string, string> | [string, string][];
```

Comment:
An observation, not a change request: in policy management, not consistently using different types for external (response) type and internal (converted) type allowed errors to appear later at inconvenient times when TypeScript or typescript-eslint linter rules became stricter after updates.